nix/tullia.nix

```diff
@@ -14,29 +14,35 @@ the rule that describes when a Tullia task is to be invoked.
 Learn more: https://github.com/input-output-hk/cicero
 */
 
-self: system:
-
 let
   ciInputName = "GitHub event";
-  repo = "input-output-hk/cardano-node";
+  repository = "input-output-hk/cardano-node";
 in rec {
   tasks = let
-    common = {
-      config,
-      ...
-    }: {
+    mkTask = top: {config, lib, ...}: {
       preset = {
-        # needed on top-level task to set runtime options
         nix.enable = true;
 
-        github-ci = {
+        github.status = {
           enable = config.actionRun.facts != {};
-          inherit repo;
-          sha = config.preset.github-ci.lib.getRevision ciInputName null;
-          clone.enable = false;
+          inherit repository;
+          revision = config.preset.github.lib.readRevision ciInputName null;
         };
       };
 
+      command.text = let
+        flakeUrl = ''github:${repository}/"$(${lib.escapeShellArg config.preset.github.status.revision})"'';
+      in config.preset.github.status.lib.reportBulk {
+        bulk.text = ''
+          echo '["x86_64-linux", "x86_64-darwin"]' | nix-systems -i \
+          | jq 'with_entries(.key |= {"x86_64-linux": "linux", "x86_64-darwin": "macos"}[.])'
+        '';
```

Comment:
I _do_ think we want to retain the architecture? We'll have aarch64-darwin in the system soon, and we'll need to distinguish between both.

Reply:
At the moment there's only `.#hydraJobs.{linux,macos}`. I guess aarch64-darwin would be added as another key like `macos-m1`. We can update this later when that key is added.

IMHO we should ideally just have `.#hydraJobs.{x86_64-linux,x86_64-darwin,aarch64-darwin}` but that is out of scope of this PR.